### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by xeeshanAqovia**

### CODE REVIEW

**Review by taliesins — COMMENTED** [review]
We will only change when Nimbus 3 supports at least once delivery. So we probably want to add that to this list of things preventing is from using 3.

Might be nice to have a bullet list what makes our fork special!

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Nimbus
 Nimbus is a .NET client library to add an easy to develop against experience against the Azure Service Bus brought to you by Damian Maclennan and Andrew Harcourt
 
+This repository is forked from [Nimbus API](https://github.com/NimbusAPI), bringing additional functionality on top of the functionality provided by Nimbus. This forkmaster branch is the default branch, and is still based on [Nimbus V2](https://github.com/NimbusAPI/Nimbus_v2), since version 3 has introduced changes such as at-most-once delivery guarantees, as opposed to at-least-once delivery guarantees.
```
Comment (slawomir-brzezinski-at-interxion) [inline readme.md]: > This repository is ... bringing additional functionality

Can we please list at least one example of such additional functionality to justify the existence of this repository?

If all that we add are bug fixes, then I would reword this not to mention 'functionality' and also list at least one bugfix with a PR to [Nimbus V2](https://github.com/NimbusAPI/Nimbus_v2), to be able to track whether the fork is still needed.

Comment (ma499) [reply readme.md]: I suggest rather than listing, we add a filter link to PRs into Nimbus from the Aqovia fork. We raise an upstream PR for every enhancement so this will then be a living list (and a checklist for eventual closure of the fork).

**Review by ma499 — APPROVED** [review]
I'm happy with the Readme now. Paired with Ricky on latest changes.

**Review by slawomir-brzezinski-at-interxion — APPROVED** [review]
